onnxruntime/core/providers/migraphx/migraphx_execution_provider.cc

```diff
@@ -1191,6 +1198,10 @@ Status MIGraphXExecutionProvider::Compile(const std::vector<FusedNodeAndGraph>&
 
         migraphx::compile_options co;
         co.set_fast_math(false);
+        co.set_exhaustive_tune_flag(false);
```

Comment:
nit: may simplified as `co.set_exhaustive_tune_flag(exhaustive_tune_)`;

Reply:
Not nit. All good

Reply:
Done